Readme.md

```diff
@@ -178,6 +178,9 @@ class FooTestCase(TestCase):
         # ...
 ```
 
+## Naming
+
+Use CapitalizedCamelCase for class names, snake_case for everything else. Use CAPITALIZED_SNAKE_CASE for global consts. See [Google Style Guide](https://google.github.io/styleguide/pyguide.html#Naming) for more.
```

Comment:
I maintain that "See Google Style Guide" (with Google Style Guide linked and capitalized) is correct, but given that we have 2 other occurrences of "the Google style guide" (lower-cased, I will note) I will change it in the interest of consistency ;-)